packages/framework/client-logger/external-telemetry/src/factory/index.ts

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IContainer } from "@fluidframework/container-definitions";
+import { ContainerTelemetryManager } from "../container";
+import { ContainerEventTelemetryProducer } from "../container/telemetryProducer";
+import type { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { AppInsightsTelemetryConsumer } from "../common";
+
+/**
+ * Namespace for grouping things related to {@link (TelemetryManagerConfig:interface)}
+ *
+ * @beta
+ */
+export namespace TelemetryManagerConfig {
```

Comment:
We strongly discourage the use of namespaces in the repo, is it really necessary for any reason? Or could we just have the bare interfaces/types/consts? (But also, see another comment that would result in the whole namespace just disappearing).

Reply:
I've reworked the config file and removed the namespace

I originally used a namespace to simplify some of the lengths of these type names and ideally make it easier to discover the type.